projects/standard-rulesets/src/examples/requireValidExamples.ts

```diff
@@ -7,10 +7,36 @@ import {
 } from '@useoptic/rulesets-base';
 import { OpenAPIV3 } from 'openapi-types';
 import Ajv, { SchemaObject } from 'ajv';
+import addFormats from 'ajv-formats';
 
-const ajv = new Ajv({ strict: false });
+const ajv = (() => {
+  const validator = new Ajv({ strict: false });
+  addFormats(validator);
+  // override pattern keyword when invalid regex
+  validator.removeKeyword('pattern');
+  validator.addKeyword({
+    keyword: 'pattern',
+    type: 'string',
+    schemaType: 'string',
+    error: {
+      message: (cxt) => {
+        return `pattern not matched ${cxt.schema}`;
+      },
+    },
+    compile: (pattern) => {
+      let regex;
+      try {
+        regex = new RegExp(pattern);
+      } catch (e) {
+        return (data) => true;
+      }
```

Comment:
Is there any way we report that the format / pattern is invalid? I'm not sure how we would do this but it feels confusing to have formats typoed and not validating silently

Reply:
@niclim I agree we should have a way to allow warnings to come out of rules, esp as more and more teams write their own. It's useful to distinguish problem with ruleset execution from problem with API. 

I feel like this is a reasonable (documented) compromise and is ok for now, but should be part of a larger conversation about the warnings.